Source/Core/VideoBackends/D3D/DXTexture.cpp

```diff
@@ -87,7 +88,8 @@ bool DXTexture::CreateSRV()
                                   D3D11_SRV_DIMENSION_TEXTURE2DARRAY,
       D3DCommon::GetSRVFormatForAbstractFormat(m_config.format), 0, m_config.levels, 0,
       m_config.layers);
-  HRESULT hr = D3D::device->CreateShaderResourceView(m_texture.Get(), &desc, &m_srv);
+  HRESULT hr =
```

Comment:
IIRC `CreateSRV()` shouldn't be called more than once, so this could probably be replaced with
```
DEBUG_ASSERT(!m_srv);
HRESULT hr = D3D::device->CreateShaderResourceView(m_texture.Get(), &desc, m_srv.GetAddressOf());
```

We probably should put these in the constructor anyway, but that's a larger change...

Reply:
Probably. I can add an assertion just to be sure.